attributes/web.rb

```diff
@@ -0,0 +1,10 @@
+#
+# Cookbook Name:: stackstorm
+# Attributes:: web
+#
+
+# Self-signed certificate
+default['stackstorm']['ssl']['self_signed']['enabled'] = false
```

Comment:
Does `['stackstorm']['ssl']['self_signed']['...']` --> `['stackstorm']['web']['ssl']['self_signed']['...']` sound right? Or do you prefer some different hierarchy?